cluster/src/main/java/com/hartwig/pipeline/tertiary/amber/Amber.java

```diff
@@ -45,11 +48,58 @@ public String namespace() {
 
     @Override
     public List<BashCommand> tumorReferenceCommands(final SomaticRunMetadata metadata) {
-        return List.of(AmberCommandBuilder.newBuilder(resourceFiles)
-                .tumor(metadata.tumor().sampleName(), getTumorBamDownload().getLocalTargetPath())
-                .reference(metadata.reference().sampleName(), getReferenceBamDownload().getLocalTargetPath())
-                .targetRegionsBed(resourceFiles.targetRegionsBed())
-                .build(), new CopyResourceToOutput(resourceFiles.amberSnpcheck()));
+
+        List<String> arguments = Lists.newArrayList();
+
+        arguments.add(String.format("-tumor %s", metadata.tumor().sampleName()));
```

Comment:
I don't see an improvement here by removing the builder. It's a little more straightforward this way, but reduces but introduces a bunch of duplication and reduces encapsulation.